openApiDocs/beta/DirectoryObjects.yml

```diff
@@ -599,31 +569,12 @@ components:
         minimum: 0
         type: integer
       example: 50
-    skip:
-      name: $skip
-      in: query
-      description: Skip the first n items
-      schema:
-        minimum: 0
-        type: integer
     search:
       name: $search
       in: query
       description: Search items by search phrases
       schema:
         type: string
-    filter:
```

Comment:
This is a breaking change, see https://github.com/microsoftgraph/microsoft-graph-devx-api/issues/611.